intellij/.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# Dependabot configuration:
```

Comment:
This comment applies to all files in the `intellij/.github/*` directory. These files look like they came from the template repo. There can only be one .github directory in a repo.

We will want to have workflows like these at some point. For now, we can remove these.